### PR TITLE
Image load clarification

### PR DESCRIPTION
**Pull request by vicloginov**

### CODE REVIEW

**Review by aniktash — COMMENTED** [review]
sounds good

**Review by rotx-maxim — APPROVED** [review]

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
You are changing all the working Unix openocd command lines to Windows versions. Could we have both please?

**Comment by vicloginov** [comment]
I will fix it

File: Examples/MAX78000/CNN/rps-demo/README.md [hunk 1]
```diff
@@ -124,7 +136,7 @@ To create your own header file follow these steps:
 
 1.  Navigate to Utility directory. $ cd Utility
 2.  Download rock, paper or scissors hand gesture image in this directory.
-3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('./image\_filename.format')). Save the cahanges.
+3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('./image\_filename.format')). Save the changes.
```
Comment (aniktash) [inline Examples/MAX78000/CNN/rps-demo/README.md:139]: Image.open('image_filename.format')

**Review by aniktash — COMMENTED** [review]
looks ok

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**